src/scitokens/scitokens.py

```diff
@@ -215,28 +215,31 @@ def _get_issuer_publickey(header, payload, insecure=False):
         # Get the issuer
         issuer = payload['iss']
         
+        # Set the user agent so Cloudflare isn't mad at us
+        headers={'User-Agent': 'Mozilla/5.0'}
```

Comment:
Seems like a questionable U-A.  Isn't there one for curl or python or something?

Reply:
The urllib/python User-Agent is default, and it was being blocked by cloudflare (thinking it's a bot).  Curl works though.  Would a curl user agent be better?  Or should we make our own user-agent?

Here's an example curl user-agent:
```
User-Agent: curl/7.52.1
```

Reply:
Let's do our own user agent, including the library version.